packages/wallet-sdk/src/core/message/RPCMessage.ts

```diff
@@ -14,10 +14,16 @@ export type EncryptedData = {
   cipherText: ArrayBuffer;
 };
 
+export type MobileEncryptedData = {
```

Comment:
what's this?

Reply:
whoops was added during a bad stash apply - removed [8d488f4](https://github.com/coinbase/coinbase-wallet-sdk/pull/1392/commits/8d488f4103a32ba6f86cbe16b2aaaf60dfaa0228)[8d488f4](https://github.com/coinbase/coinbase-wallet-sdk/pull/1392/commits/8d488f4103a32ba6f86cbe16b2aaaf60dfaa0228)[8d488f4](https://github.com/coinbase/coinbase-wallet-sdk/pull/1392/commits/8d488f4103a32ba6f86cbe16b2aaaf60dfaa0228)